docs/release-notes/8.8.asciidoc

```diff
@@ -0,0 +1,90 @@
+[[release-notes-header-8.8.0]]
+== 8.8
+
+[[release-notes-8.8.0]]
+=== 8.8.0
+
+[discrete]
+[[known-issue-8.8.0]]
+==== Known issues
+
+* Setting the `max_signals` value higher than the {kibana-ref}/alert-action-settings-kb.html#alert-settings[`xpack.alerting.rules.run.alerts.max`] value will lead to rule failure.
+
+[discrete]
+[[breaking-changes-8.8.0]]
+==== Breaking changes
+
+//tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+There are no breaking changes in 8.8.0.
+//end::breaking-changes[]
+
+
+[discrete]
+[[deprecations-8.8.0]]
+==== Deprecations
+There are no deprecations in 8.8.0.
+
+
+[discrete]
+[[features-8.8.0]]
+==== New features
+
+* Adds a new response action that allows you to execute commands on a selected host (https://github.com/elastic/security-team/issues/5441[5441]).
```

Comment:
> @joepeeples @kevinlog should this link to a specific PR, or a set of PRs, instead of the dev epic?

Yes, also the `security-team` repo is private, so that link would produce a 404 for anyone outside of Elastic. There are tons of issues & PRs for this feature, but this one seems good for RNs:

```suggestion
* Adds a new response action that allows you to execute commands on a selected host ({pull}150202[#150202]).
```